### ISSUE TITLE
Resizing video output on Coral Dev Board with HDMI monitor

### ISSUE THREAD

**Issue by agusit7** [google-coral/project-posenet#13]
I'm trying to get the output video from the pose-camera demo to run full-screen on the Dev Board through an HDMI monitor. Is this supported? How can I modify the demo to change just the output video without increasing the input image? I'm not interested in resolution, I just want to fill the whole screen on a 4k monitor.

**Comment by jonasl** [comment]
This project is using GStreamer and rendering to a full screen window is supported and done like for any other GStreamer app.
That is, the application is responsible for creating and managing the window. It's common and convenient to use GTK for this but there are other options. If you're writing your app in C/C++ this example shows how to create and configure glimagesink in a Wayland window:
https://github.com/GStreamer/gst-plugins-base/blob/master/tests/examples/gl/gtk/gstgtk.c

Now this particular example is written in Python, using the official Python bindings for GStreamer and friends. These binding generally work well but in regards to Wayland a couple of entrypoins are sadly missing. So today, for a full screen GTK window with glimagesink, one needs to use ctypes. I've posted an example here:
https://gist.github.com/jonasl/92c1ef32cfd87047e15f5ae24c6b510e
This sample code is generic and doesn't depend on Coral.

I pushed a commit here to show how this can be used in the context of this demo:
https://github.com/jonasl/project-posenet/commit/f9c324994e12566d556cd873b78de7630a373a81

In a future version of the Coral Dev Board system image we will simplify this process with ready to use helpers. project-posenet will run full screen on the dev board by default.
